.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - conda update conda
 
   # Install dependencies
-  - conda create -n test --file requirements/run.txt python=3.8
+  - conda create -n test --file requirements/run.txt python=3.9
```

Comment:
I think we agreed that this should go to python=3, and then I can merge (also removing the pin to 3.9 above if it is not used, to avoid confusion later.